fe/fe-core/src/main/java/com/starrocks/analysis/OutFileClause.java

```diff
@@ -163,12 +163,6 @@ private void analyzeProperties() throws AnalysisException {
             }
             processedPropKeys.add(PROP_MAX_FILE_SIZE);
         }
-
-        if (processedPropKeys.size() != properties.size()) {
-            LOG.debug("{} vs {}", processedPropKeys, properties);
```

Comment:
why delete this function

Reply:
We have to support more properties for authentication.